Beta and Aurora-channel Activity Stream tests are failing due to HTTP 204 response

https://github.com/mozmeao/snippets-service/blob/ec3d4c588503517875a2f069056ce4e9617aff8c/tests/test_snippets.py#L41 has been failing for **aurora** and **beta** channels, while **release** is fine for at least the past couple days.

TL;DR: https://snippets.mozilla.com/5/Firefox/default/default/default/en-US/aurora/default/default/default/ and https://snippets.mozilla.com/5/Firefox/default/default/default/en-US/beta/default/default/default/ serve up HTTP 204s with 0-byte text/html content (see https://www.webpagetest.org/result/171129_AD_1ba3f01647797beeb2791f8855e6d72a/1/details/#waterfall_view_step1 vs. the working _release:_ 

```
00:18:07 =========================== short test summary info ============================
00:18:07 FAIL test_snippets.py::test_activity_stream[aurora]
00:18:07 FAIL test_snippets.py::test_activity_stream[beta]
00:18:07 =================================== FAILURES ===================================
00:18:07 _________________________ test_activity_stream[aurora] _________________________
00:18:07 [gw7] linux2 -- Python 2.7.5 /home/jenkins/slave/workspace/snippets.prod/tests/.tox/py27/bin/python2.7
00:18:07 
00:18:07 base_url = 'https://snippets.mozilla.com', channel = 'aurora'
00:18:07 
00:18:07     @pytest.mark.parametrize(('channel'), ['aurora', 'beta', 'release'])
00:18:07     def test_activity_stream(base_url, channel):
00:18:07         url = URL_TEMPLATE.format(base_url, '5', channel)
00:18:07         soup = _parse_response(_get_redirect(url).content)
00:18:07 >       script = soup.find('script', type='application/javascript').text
00:18:07 E       AttributeError: 'NoneType' object has no attribute 'text'
00:18:07 
00:18:07 base_url   = 'https://snippets.mozilla.com'
00:18:07 channel    = 'aurora'
00:18:07 soup       = 
00:18:07 url        = 'https://snippets.mozilla.com/5/Firefox/default/default/default/en-US/aurora/default/default/default/'
00:18:07 
00:18:07 test_snippets.py:44: AttributeError
Post stage
00:18:07 __________________________ test_activity_stream[beta] __________________________
00:18:07 [gw1] linux2 -- Python 2.7.5 /home/jenkins/slave/workspace/snippets.prod/tests/.tox/py27/bin/python2.7
00:18:07 
00:18:07 base_url = 'https://snippets.mozilla.com', channel = 'beta'
00:18:07 
00:18:07     @pytest.mark.parametrize(('channel'), ['aurora', 'beta', 'release'])
00:18:07     def test_activity_stream(base_url, channel):
00:18:07         url = URL_TEMPLATE.format(base_url, '5', channel)
00:18:07         soup = _parse_response(_get_redirect(url).content)
00:18:07 >       script = soup.find('script', type='application/javascript').text
00:18:07 E       AttributeError: 'NoneType' object has no attribute 'text'
00:18:07 
00:18:07 base_url   = 'https://snippets.mozilla.com'
00:18:07 channel    = 'beta'
00:18:07 soup       = 
00:18:07 url        = 'https://snippets.mozilla.com/5/Firefox/default/default/default/en-US/beta/default/default/default/'
00:18:07 
00:18:07 test_snippets.py:44: AttributeError
00:18:07 ===================== 2 failed, 11 passed in 2.82 seconds ======================
```

Failure log from console: https://gist.github.com/stephendonner/a8234ff3ccee6952d1c5fcf8d48a1c7c

/cc @davehunt and @glogiotatidis

--- a/tests/test_snippets.py
+++ b/tests/test_snippets.py
@@ -37,9 +37,8 @@ def test_response_codes(base_url, version, channel):
     assert r.status_code in (requests.codes.ok, requests.codes.no_content)
 
 
-@pytest.mark.parametrize(('channel'), ['aurora', 'beta', 'release'])
-def test_activity_stream(base_url, channel):
-    url = URL_TEMPLATE.format(base_url, '5', channel)
+def test_activity_stream(base_url):
+    url = URL_TEMPLATE.format(base_url, '5', 'release')
     soup = _parse_response(_get_redirect(url).content)
     script = soup.find('script', type='application/javascript').text
     snippet_json_string = re.search('JSON\.parse\("(.+)"\)', script).groups()[0]

